silx/gui/plot/actions/io.py

```diff
@@ -144,9 +145,11 @@ class SaveAction(PlotAction):
     SCATTER_FILTER_NXDATA = 'Scatter as NXdata (%s)' % _NEXUS_HDF5_EXT_STR
     DEFAULT_SCATTER_FILTERS = (SCATTER_FILTER_NXDATA,)
 
+    IMAGE_STACK_FILTER_NXDATA = 'Stack of images as NXdata (%s)' % _NEXUS_HDF5_EXT_STR
+
     # filters for which we don't want an "overwrite existing file" warning
     DEFAULT_APPEND_FILTERS = (CURVE_FILTER_NXDATA, IMAGE_FILTER_NXDATA,
-                              SCATTER_FILTER_NXDATA)
+                              SCATTER_FILTER_NXDATA, IMAGE_STACK_FILTER_NXDATA)
```

Comment:
It would be best to define `IMAGE_STACK_FILTER_NXDATA` in `ScatterView` and provide a way to extend the `DEFAULT_APPEND_FILTERS` from outside `SaveAction`.

One way is to add an argument to `setFileFilter` (e.g., `overwriteWarning=True`) and to use a list attribute (initialised with the content of `DEFAULT_APPEND_FILTERS` to check for dialog that needs overwrite warning.

Reply:
Yes perfect, how about calling the argument `append` ?

Reply:
`append` is not meaningful to me, without ready the doc, I'd expect it to "append" the filter to the end of the list of filter....
`allowOverwite` or `withOverwrite`?

Reply:
or `appendToFile`?

Reply:
Any other options?

Reply:
`appendToFile` is good